pkg/monitor/api/drop.go

```diff
@@ -85,6 +85,7 @@ var errors = map[uint8]string{
 	187: "L3 translation from IPv4 to IPv6 failed (NAT46)",
 	188: "L3 translation from IPv6 to IPv4 failed (NAT64)",
 	189: "Authentication required",
+	190: "DSR with GENEVE not supported for IPv6 with XDP",
```

Comment:
I have a mild concern here that this is too specific of a drop code. If we allocated DROP codepoints for every four features that don't work together, we would exceed the remaining number space.

The drop message already contains the information that this is IPv6. I wonder if we should just make this something like:

```suggestion
	190: "Unsupported packet protocol for DSR encapsulation",
```

At least then if we ever had other encap protocols or problems outside XDP, we could use the same drop code.